examples/angular_bazel_architect/projects/frontend-lib/BUILD.bazel

```diff
@@ -78,10 +78,7 @@ architect_test(
 )
 
 pkg_npm(
-    name = "npm_package",
+    name = "frontend-lib",
     package_name = "frontend-lib",
-    # This is a workaround for the Linker When using `output_dir = True`.
-    # the ':build' target dependency has an output directory so it will end up at 'npm_package/build' instead of
-    # the root of the pkg_npm output directory.
-    nested_packages = [":build"],
+    deps = [":build"],
```

Comment:
kinda strange that it's `deps` feels like it wants to be `src` WDYT?

Reply:
It should probably work with both. Not sure which is better in this case as `:build` refers to the target `:build` but it is also the name of the directory artiract.